data/sql/updates/pending_db_world/rev_1689021841397941900

```diff
@@ -0,0 +1 @@
+UPDATE `quest_template_addon` SET `RequiredSkillPoints` = 70 WHERE (`ID` = 1582);
```

Comment:
it's missing the .sql in the file name

you probably don't see it properly for a Windows settings that you could remove following this https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/How-to-enable-hidden-file-extensions-in-Windows.html